tools/report-converter/README.md

```diff
@@ -0,0 +1,46 @@
+# CodeChecker Report Converter
+
+A Python tool to parse and convert various code analyzer outputs into CodeChecker format. It can create HTML reports from analyzer outputs and store them in a CodeChecker server.
+
+## Quick Start
+
+```bash
+# Create a Python virtualenv and set it as your environment
+make venv
+source $PWD/venv/bin/activate
+
+# Build and install report-converter package
+make package
+```
+
+## Basic Usage
+
+```bash
+# Convert analyzer output to CodeChecker format
+report-converter --type clang-tidy --output /path/to/reports clang-tidy-output.txt
+
+# Generate HTML report
+plist-to-html --input /path/to/reports/*.plist --output /path/to/html
```

Comment:
I can't find this `plist-to-html` script.

Reply:
There is a entry point defined for the report-converter package, which is named `plist-to-html`:
/tools/report-converter/report_converter.egg-info/entry_points.txt
This is a way for Python packages to provide a CLI script once installed.
I just went ahead and included this one in the docs as well.

Reply:
Maybe, that entry point should be removed from setup.py, because it really doesn't exist. Or, am I doing something wrong?